lib/whmcs/base.rb

```diff
@@ -34,18 +40,20 @@ def self.send_request(params = {})
       req.set_form_data(params)
 
       res = http.start { |http| http.request(req) }
-      parse_response(res.body)
+      parse_response(HTMLEntities.new.decode(res.body))
```

Comment:
I think the `HTMLEntities` call should probably occur in `#parse_response` - feels like something that is part of the parsing